src/app/views/data/data.js

```diff
@@ -30,6 +54,28 @@ class DataViewController extends ADBMapController {
   _addGeoJSONLayer(geojson) {
     L.geoJson(geojson).addTo(this.map);
   }
+
+  _onResize() {
+    this.charts.width = document.querySelectorAll(".histogram .panel-content p")[0].clientWidth;
+    try {
```

Comment:
Because the $imeout runs on the next js interpreter tick, if changing this makes the redraw feel sluggish, then I think it can be safely left as is.